wikidataintegrator/wdi_helpers/publication.py

```diff
@@ -548,12 +551,44 @@ def biorxiv_api_to_publication(biorxiv_id: str, id_type='biorxiv') -> Publicatio
     return publication
 
 
+def chemrxiv_api_to_publication(chemrxiv_id: str, id_type='chemrxiv') -> Publication:
```

Comment:
In theory, this could be used for other figshare *Rxiv servers as well, but the JSON response doesn't give any hints as to which database the result is actually part of